src/components/Table/TagSearch/index.vue

```diff
@@ -320,15 +320,15 @@ export default {
       // this.$emit('tagSearch', this.filterMaps)
 
       // 修改查询参数时改变url中保存的参数
-      if (this.getUrlQuery) {
-        let newQuery = _.cloneDeep(this.$route.query)
-        if (this.filterKey.startsWith('search')) {
-          newQuery = { ...newQuery, search: encodeURI(this.filterMaps.search) }
-        } else {
-          newQuery = { ...newQuery, [this.filterKey]: encodeURI(this.filterValue) }
-        }
-        this.$router.replace({ query: newQuery })
-      }
+      // if (this.getUrlQuery) {
+      // let newQuery = _.cloneDeep(this.$route.query)
+      // if (this.filterKey.startsWith('search')) {
+      // newQuery = { ...newQuery, search: encodeURI(this.filterMaps.search) }
+      // } else {
+      // newQuery = { ...newQuery, [this.filterKey]: encodeURI(this.filterValue) }
+      // }
+      // this.$router.replace({ query: newQuery })
+      // }
 
       this.filterKey = ''
       this.filterValue = ''
```

Comment:
There aren't any known issues with the provided code snippet, it appears to be functioning as expected based on what is written here. If you want me to take a closer look at certain specific areas like function names or variable usage, please specify them, I'd still suggest maintaining this line of inquiry for further development purposes.